CHANGELOG.md

```diff
@@ -838,6 +838,9 @@ Non-backwards compatible changes
 
   4. The modules `Relation.Nullary.(Product/Sum/Implication)` have been deprecated
          and their contents moved to `Relation.Nullary.(Negation/Reflects/Decidable)`.
+		 
+  5. The proof `T?` has been moved from `Data.Bool.Properties` to `Relation.Nullary.Decidable.Core`
+	 (but is still re-exported by the former).
```

Comment:
Is the re-export, as opposed to an explicit deprecation here, th best choice? Eg `Data.Nat.Properties` imports *only* `T?` from `Data.Nat.Properties` (and presumably also some of the downstream uses of `Decidable` relations in eg your work on #2186 

I guess my point in all my comments on this topic is that, if you are going to the trouble of importing `Relation.*.Decidable`, then it's an unnecessary detour to re-import from `Data.Bool.Properties` structure which would (after the present PR is merged) be already present...
... and this seems to be for the sake of ... well I'm not sure exactly?

Reply:
I think my view is that `T` is defined in `Data.Bool.Base` and therefore we should export the property that it is decidable from `Data.Bool.Properties`. The fact that dependency cycles force us to actually prove it earlier in the dependency graph outside of the `Data.Bool` folder is largely irrelevant.

Reply:
OK!